.editorconfig

```diff
@@ -3,5 +3,5 @@ indent_style=space
 indent_size=2
 tab_width=2
 end_of_line=lf
-insert_final_newline=false
-charset=utf-8
+insert_final_newline=true
```

Comment:
this is deliberate as the previous config contradicts with prettier which hard coded to insert a new line at the end